### PR TITLE
Add support for webpack dynamic imports in scripts

### PR DESCRIPTION
**Pull request by ripvannwinkler**
Webpack 2.0+ includes support for an import() statement to dynamically load functions. Babylon has been updated to account for this but the plugin had to be added to the config. I've done so and added a test for it.

Also, removed an extraneous line break that was causing an otherwise valid pot extraction test to fail.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-gettext-parser",
+  "name": "@elser/react-gettext-parser",
```
Comment (alexanderwallin) [inline package.json]: This will have to be reverted. 😊

File: tests/fixtures/SingleString.pot [hunk 2]
```diff
@@ -1,3 +1,2 @@
-
```
Comment (alexanderwallin) [inline tests/fixtures/SingleString.pot]: When did this cause errors?

File: tests/tests.js [hunk 3]
```diff
@@ -116,6 +116,10 @@ describe('react-gettext-parser', () => {
       expect(messages).to.have.length(1);
     });
 
+    it('should not fail on import()', () => {
+      const messages = extractMessagesFromFile('tests/fixtures/DynamicImportCaller.js');
```
Comment (alexanderwallin) [inline tests/tests.js]: Could this be more affirmative, like, `supports import()` and then expecting messages to be something?

File: tests/fixtures/SingleString.pot (hunk 2 above)
Comment (ripvannwinkler) [inline tests/fixtures/SingleString.pot]: 2) react-gettext-parser compiling to pot should spit out a pot file containing an extracted string:

	      AssertionError: expected 'msgid "Translate me"\nmsgstr ""' to equal 'msgid "Translate me"\r\nmsgstr ""'
	      + expected - actual

	      -msgid "Translate me"
	      +msgid "Translate me"
	       msgstr ""

File: tests/tests.js (hunk 3 above)
Comment (ripvannwinkler) [inline tests/tests.js]: Surely can adjust the description, but I'm not sure the actual import() statement can be tested via a message assertion. It's only a stage 3 proposal for now (requires babel to use). For this specific case, it should be sufficient test-wise to verify that the babylon parser doesn't choke on the syntax. You can remove the babylon plugin `dynamicImport` in the defaults file to see what I'm talking about.

File: package.json (hunk 1 above)
Comment (ripvannwinkler) [inline package.json]: Sorry, everything after commit e1af58f should not be excluded in the pull request.  should have made those changes on another branch. I'll resubmit the pull request on a different branch once you take a look at my replies to the other related comments.

Comment (alexanderwallin) [inline package.json]: You can `reset --hard` to that commit and force push it up here too, if you want. Nice to keep a record, but no biggie!

File: tests/fixtures/SingleString.pot (hunk 2 above)
Comment (alexanderwallin) [inline tests/fixtures/SingleString.pot]: Hah! `\n` versus `\r\n`! Are you on a Windows machine?

File: tests/tests.js (hunk 3 above)
Comment (alexanderwallin) [inline tests/tests.js]: Fair enough!

File: tests/fixtures/SingleString.pot (hunk 2 above)
Comment (ripvannwinkler) [inline tests/fixtures/SingleString.pot]: Haha yep. I'll revert/ignore this one then. Silly carrige returns.

**Pull request closed without merging**